tests/utils/json/tests.py

```diff
@@ -31,4 +33,12 @@ def __repr__(self):
                 return 'Unknown object'
 
         obj = Unknown()
-        assert json.dumps(obj) == '"Unknown object"'
\ No newline at end of file
+        assert json.dumps(obj) == '"Unknown object"'
+
+    def test_decimal(self):
+        d = {'decimal': Decimal('123.45')}
+        assert json.dumps(d) == '{"decimal": "123.45"}'
+
+    def test_translation(self):
+        d = {'lazy_translation': gettext_lazy('testing')}
```

Comment:
I think this might still fail with `ImproperlyConfigured` from django as no settings are configured.

Maybe needs moving to django tests?